### ISSUE TITLE
Problem with different save file sizes.

### ISSUE THREAD

**Issue by Leeful** [ChendoChap/Playstation-4-Save-Mounter#5]
I cannot transfer my save of Fallout 4.
The Comleted game save is 22.1MB but a Fresh NEW save game is only 3.39MB.

When I try and overwrite the mounted 3.39MB save with the larger 22.1MB file, only 4.65MB is transfered.

On my FTP client it says the transfer was successful but only 4.65MB is actually transfered.

Could you please look into this issue.
Thanks.

**Comment by ChendoChap** [comment]
I'm not entirely sure, afaik saves are created with a max size.
so saves shouldn't be larger than what the game specifies upon creating a save.
You're sure you didn't replace it with the wrong files or mounted a different save (eg settings save)?

**Comment by Leeful** [comment]
Thanks for the reply.
I started a new game to check the file size and as you can see from the attached screenshot the new game is 6.39MB but the OLD completed game is 29.95MB.
![20181205225109](https://user-images.githubusercontent.com/17101646/49549547-faddc200-f8e0-11e8-90dd-349f4cc87bf3.png)

**Comment by ChendoChap** [comment]
save data management doesn't show a save's max size. saves on ps4 are containers with a specified size limit. eg settings save usually doesn't need a lot of space while rpg main saves are usually larger than 5MB.
so you should have been able to replace the save (I don't now why it doesn't work). Can you send me the save? I have fallout and I can try messing around with it although that might take some time I'm quite busy atm.

**Comment by Leeful** [comment]
[Fallout_4_CUSA03450_SAVE.zip](https://github.com/ChendoChap/Playstation-4-Save-Mounter/files/2658465/Fallout_4_CUSA03450_SAVE.zip)
Here is the save. 
Thanks for looking into it. Good luck :)

**Comment by ChendoChap** [comment]
found a workaround, I was able to load your save

**Comment by Leeful** [comment]
Thats great news :)
Is it a simple workaround the end user can do or will you need to compile a new version of Save-Mounter?

**Comment by ChendoChap** [comment]
it's pretty simple, it creates new saves and mounts them.
input:
- save name (eg SDU9)
- save size in blocks (1 block = 32kb)
--
I'm not sure if I'm going to add it to Save Mounter or release it as a separate executable.

**Comment by Leeful** [comment]
Thanks, I knew you would find a way :)
Looking forward to a release.

**Comment by ChendoChap** [comment]
this should work, I'll commit this stuff later
~~[test.zip](https://github.com/ChendoChap/Playstation-4-Save-Mounter/files/2688200/test.zip)~~

**Comment by Leeful** [comment]
I just tested this and there is good new and bad news.
The good news is that I could load the 22MB save on my other PS4. 
The bad news is that the dummy save it created is 1.08GB in size.

Moving the 'max save size' slider does not make any difference. The save it creates is always 1.08GB
I tried leaving it at 3MB, moving it to 25MB, 30MB but it makes no difference.

The problem now is, because the save game is 1GB and not 22MB, the game will not let any more saves be created or even overwrite the autosave.
In the game it says _"The maximum number of saves has been reached. Please delete older saves to be able to create a new one"_

It's probably just a bug in the Save Mounter exe.
I'm sure you will be able to fix it. :)

**Comment by ChendoChap** [comment]
fixed in v1.3 release. although you could have deleted the save in game while it was loaded and saved afterwards. Does it now work as it should?

**Comment by Leeful** [comment]
Everything is now working as it should in the v1.3 release version.

Thanks for all your help. :)

**Comment by crazyrokr** [comment]
> Everything is now working as it should


I extracted autosave folder with `sce_sys` and `SAVEDATA.DAT` (around 15 MB). Then I did full initialization, installed Fallout 4, created a save with name `SDU2` and size 40 MB and put only `SAVEDATA.DAT` into newly created save directory. After ftp session I press unmount. Unfortunately F4 says that data is corrupted. I've no idea what's wrong. Maybe some actions on `param.sfo` are required. Also my PS4(7.55) was neither activated before initialization nor after so console id is 0000000000... in param.sfo
